bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -168,32 +168,33 @@ then
         > "${PATRONI_POSTGRESQL_DATA_DIR}/postgresql.conf"
     fi
 
-    manual_start_pg_ctl_options=""
-    # detect if this is a server that needs to entre recovery mode, which is
-    # what might happen after an instance is cloned.
-    # if this is the case, we will want to have a warm standby
```

Comment:
Why is this comment removed?

Reply:
My understanding is that we control recovery because Patroni won't. That's probably what should be described here. We set some parameters to get the recovery behavior we want. "warm standby" encompasses recovery behavior but it wasn't clear to me. I intended for the new comment on L174 to capture this.

The concept of "cloning" seemed out of place at this layer, but that could just be me. I'll put some comments before the conditional. Is there anything you'd like to see there?

Reply:
Along the lines of what you suggest. However, the `"warm standby"` bit is important, because you can recover as a "hot standby" which will cause some behavior changes in terms of detection (unless those were also modified)